templates/tpos/tpos.html

```diff
@@ -405,29 +405,34 @@ <h5 class="q-mt-none q-mb-sm">${totalfsat}<small> sat</small></h5>
         v-if="invoiceDialog.data"
         class="q-pa-lg q-pt-xl lnbits__dialog-card"
       >
-        <q-responsive :ratio="1" class="q-mx-xl q-mb-md">
-          <lnbits-qrcode
-            :value="'lightning:' + invoiceDialog.data.payment_request.toUpperCase()"
-            :options="{width: 800}"
-            class="rounded-borders"
-          ></lnbits-qrcode>
-        </q-responsive>
+        <a
+          :href="'lightning:' + invoiceDialog.data.payment_request.toUpperCase()"
```

Comment:
nitpick: put this expression into a `computed` method (used in two places):
`'lightning:' + invoiceDialog.data.payment_request.toUpperCase()`